models/preles/R/runPRELES.jobsh.R

```diff
@@ -52,43 +52,54 @@ runPRELES.jobsh<- function(met.file,outdir,parameters, sitelat, sitelon,start.da
   ifelse(leap_year(year)==TRUE,
          dt <- (366*24*60*60)/length(sec), #leap year
          dt <- (365*24*60*60)/length(sec)) #non-leap year
-  tstep = 86400/dt
+  tstep = round(86400/dt)
+  dt = 86400/tstep
   
-  doy <- rep(1:365,each=86400/dt)
+  doy <- rep(1:365,each=86400/dt)[1:length(sec)]
   if(year %% 4 == 0){  ## is leap
-    doy <- rep(1:366,each=86400/dt)
+    doy <- rep(1:366,each=86400/dt)[1:length(sec)]
   }
   
 
   ## Get variables from netcdf file
-  PAR <-ncvar_get(nc,"surface_downwelling_photosynthetic_photon_flux_in_air") #PAR in mol/m2s1
+  SW <-ncvar_get(nc,"surface_downwelling_shortwave_flux_in_air") #SW in W/m2
   Tair <-ncvar_get(nc,"air_temperature")#air temperature in K
   Precip <-ncvar_get(nc,"precipitation_flux")#precipitation in kg/m2s1
-  CO2 <-ncvar_get(nc,"mole_fraction_of_carbon_dioxide_in_air") #mol/mol
+  CO2 <-try(ncvar_get(nc,"mole_fraction_of_carbon_dioxide_in_air")) #mol/mol
```

Comment:
This is done in other places as well, if CO2 does not exist, it will compute it later, in this case see https://github.com/tonygardella/pecan/blob/master/models/preles/R/runPRELES.jobsh.R#L75